openerp/addons/base/tests/test_base.py

```diff
@@ -441,26 +441,6 @@ def test_101_create_translated_record(self):
         fr_context_cat = self.res_category.browse(cr, uid, self.cat_id, context={'lang':'fr_FR'})
         self.assertEqual(fr_context_cat.name, 'Clients', "Translation not found")
 
-    def test_102_duplicate_record(self):
```

Comment:
Same as above here also we making copy of the category and checks that duplication did set untranslated value or not.
So now we are not allowed to create duplicate category so i remove this tests as no more needed.